static/js/main.js

```diff
@@ -0,0 +1,6 @@
+
+
+$(document).ready(function () {
```

Comment:
you're setting the height of the header to be the height of the window? that does not sound right...

An any case such things are better done with CSS.

Reply:
OK,I did it! :)